### ISSUE TITLE
Hi I have a few question ! 

### ISSUE THREAD

**Issue by Yoontae6719** [thuml/Autoformer#17]
First of all, thank you for such an excellent study.
The results of the Autoformer are awe-inspiring and insightful.

My question is:
1. I wonder why the autocorrelation result is not [-1,1]. If you look at the result of corr = torchfft.irfft(res, dim = -1) in Autocorrelation.py, there are also values ​​over 10,000. I wonder why this is.

2. I am curious about the method for circular padding in nn.Conv1d(in_channels=c_in, out_channels=d_model, kernel_size=3, padding=padding, padding_mode='circular', bias=False). I'd appreciate it if you could explain.

**Comment by wuhaixu2016** [comment]
Hi, thanks for your interest. Good questions!

(1) We adopt the **unnormalized** version of autocorrelation (see equation 5 in the main paper), which is commonly used in signal processing. Note that we mainly adopt autocorrelation to find out the period length. Thus, the relative value size is essential. As for the absolute value size, we adopt the softmax for the afterward normalization.

(2) We do not consider a lot in this operation. We use circular padding only because we want to keep the series length unchanged after this projection.

**Comment by wuhaixu2016** [comment]
See no one has problems on this issue. I would like to close it.
